db/migrate/20150910153517_add_full_name_to_custom_attributes.rb

```diff
@@ -0,0 +1,5 @@
+class AddFullNameToCustomAttributes < ActiveRecord::Migration
+  def change
+    add_column :custom_attributes, :full_name, :text
```

Comment:
I guess the source can stay as it is, it is marking a 'source of truth' it comes from :-)

The unique name is nice in drift, so you immediately see, what attr it is and it's nicely sorted by file_name. Also drift needs something unique for comparing (unless I want to modify the behavior)